_docs/di/container.md

```diff
@@ -76,6 +76,77 @@ DI.set(NewProvider, 'NewProvider');
 DI.get('Provider').test(); // prints "hotdog";
 ```
 
+### Scope
+
+It is possible to manage dependencies within different scopes, and so within different [design patterns](https://en.wikipedia.org/wiki/Design_Patterns).
+
+| Scope             | Description                                                                                                                                    |
+|-------------------|------------------------------------------------------------------------------------------------------------------------------------------------|
+| `default`         | The dependency is a `singleton` and it will be resolved as so without any setting while registering a dependency or injecting via `@Inject()`. |
+| `non-singleton`   | The dependency will be resolved as a brand new instance.                                                                                       |
+
+### Register
```

Comment:
We already have a `### Register` paragraph above, maybe we could review the structure and use a four hashes heading with another title here.

Reply:
But it is contextual of Scope, I don't know it is a real issue.